scan.rb

```diff
@@ -50,12 +67,13 @@ def get_option(key)
     next if ignore_http_codes.include?(response.code.to_i)
     next if ignore_content_length > 0 && ignore_content_length == response['content-length'].to_i
 
-    puts "Found: #{hostname} (#{response.code})"
+    result << "Found: #{hostname} (#{response.code})\n"
     response.to_hash.each do |header, values|
-      puts "  #{header}:"
+      result << "  #{header}:"
       values.each do |value|
-        puts "    #{value}"
+        result<< "\n    #{value}\n"
       end
     end
   end
 end
+write_results(result.string)
```

Comment:
Can you add a new line at the end and also add a `puts result.string`? Thanks!

Reply:
fixed